tileview/src/main/java/com/qozix/tileview/TileView.java

```diff
@@ -772,7 +772,9 @@ public void onPanEnd( int x, int y, Origination origin ) {
 
   @Override
   public void onZoomBegin( float scale, Origination origin ) {
-    mDetailLevelManager.lockDetailLevel();
+    if( !Origination.DOUBLE_TAP.equals( origin ) ) {
```

Comment:
have you read my comment in #259, saying that changing detail level while zooming do not cause OOM?
If you think about it, if we allow rendering while panning, why should we forbid rendering while zooming?